### DISCUSSION TITLE
V-for Vuex

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#563] [Q&A] [answered]
Buena tardes para imprimir el arreglo de sabores que está en el state. Utilice v-for  pero no me imprime nada, me imagino que hay que colocar un get del store. 

Esa es mi duda

**Comment by CesarMezaB** [comment, marked as answer]
Yo mostré los elementos de un arreglo de la siguiente manera a otro componente espero que te pueda servir
```
            <p
              v-for="(pastel, index) in $store.state.checkedPasteles"
              :key="index"
            >
              {{ pastel }}
            </p>
```

**Reply by GloriaCoralCerecedo** [reply]
Oh ya si en un rato lo intento, algo así estaba haciendo. Gracias!

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias me funciono!

**Comment by HugoHerreraR** [comment]
si hazlo como el compañero dijo pero si es un arreglo de objetos en la parte de la impresión defines que parte, por ejemplo si tienes sabores y decoraciones y solo quieres mostrar los saboreés se escribiría   { { pastel.sabores } }

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias!

**Comment by GloriaCoralCerecedo** [comment]
Oigan otra pregunta, los estilos tiene que estar solo en las vistas o también puede ser en los componentes? claro utilizando el scoped

**Reply by HugoHerreraR** [reply]
Si también se puede hacer en un archivo aparte

**Reply by HugoHerreraR** [reply]
<style scoped>
 @import '../assets/css/why.css'
</style>

de esa forma puedes importar el archivo

**Reply by GloriaCoralCerecedo** [reply]
Gracias!
